src/containers/App/App.js

```diff
@@ -1,31 +1,31 @@
 import React, { PropTypes } from 'react';
-import { Router } from 'react-router';
+import { Router, hashHistory } from 'react-router';
 
 class App extends React.Component {
   static contextTypes = {
     router: PropTypes.object
   }
 
   static propTypes = {
-    history: PropTypes.object.isRequired,
     routes: PropTypes.element.isRequired
   };
 
   get content() {
```

Comment:
these getters are usually used for binding a function to an object property. `content` is more than just a property in this case, so I think it's better to make it a class method.

since you're returning JSX, how about `renderContent`?